packages/java-edition/src/mcfunction/parser/argument.ts

```diff
@@ -679,10 +679,16 @@ function resourceOrInline(category: core.FileCategory) {
 	}])
 }
 
-function selectorPrefix(): core.InfallibleParser<core.LiteralNode> {
+function selectorPrefix(ignoreInvalidPrefix: boolean): core.InfallibleParser<core.LiteralNode> {
 	return (src: core.Source, ctx: core.ParserContext): core.LiteralNode => {
 		const start = src.cursor
-		const value = src.readUntil(' ', '\r', '\n', '[')
+		let value: string
+		if (ignoreInvalidPrefix) {
+			value = src.peek(0, 2)
+			src.skip(2)
```

Comment:
i think this will do the same thing for you:

```suggestion
			value = src.read(2)
```

Reply:
`src.read()` doesn't accept any arguments, I had thought this would work as well but it just doesn't exist. Mandela effect something something...